### ISSUE TITLE
enabling addon fails in blender 2.91

### ISSUE THREAD

**Issue by nacjm** [ross-g/io_pdx_mesh#24]
Hey, I am trying to install the addon into blender but I keep running into this error when trying to enable it: 
Traceback (most recent call last):
  File "C:\Program Files\Blender Foundation\Blender 2.91\2.91\scripts\modules\addon_utils.py", line 382, in enable
    mod.register()
AttributeError: module 'io_pdx_mesh' has no attribute 'register'

Any idea what the cause might be?

**Comment by ross-g** [comment]
The cause is you're using new a version of Blender that is less than a day old! 😉 Looks like something changed with registering addons and I will look into that.

Please use 2.90 with the addon in the meanwhile.

**Comment by Tetrino** [comment]
'Course you comment on it while I'm sat on the page looking it up myself @ross-g ;)

 Relevant python API changes: https://wiki.blender.org/wiki/Reference/Release_Notes/2.91/Python_API

I think it's not liking the import trickery in __init__.py at root.

**Comment by ross-g** [comment]
Yes, it's actually a 1 line fix! The addon doesn't register correctly in 2.91 because my code checks ``sys.executable`` expecting it to be either ``Blender.exe`` or ``Maya.exe`` before booting up for either software.

From the release notes:
> sys.executable now points to the Python interpreter (instead of the Blender executable)

**Comment by Tetrino** [comment]
Figured it was something along those lines, great to hear it's a quick and dirty!

**Comment by ross-g** [comment]
Fixed by 09127bc7e450dcf2c029a16dcc0d8bdea40db850 in the bugfixes branch.
This issue will be closed once the fix is in a release.

**Comment by stirles** [comment]
@ross-g I am an artist with no experience coding, and am experiencing the same issue as the original post. I can see it says "fixed" under your comment "Fixed by 09127bc in the bugfixes branch." but I cant parse that at all. is there an easy way to put this in laymans terms for me? 
If context helps, I am an illustrator with some experience in blender, trying to contribute to a crusader kings 3 mod

**Comment by ross-g** [comment]
That's a great point 😄 in this instance it means that I've found the problem and uploaded the fix to Github. But haven't released a new "official" version to download yet.

You can download the work-in-progress version (the bugfixes branch) if you want!
https://github.com/ross-g/io_pdx_mesh/archive/bugfixes.zip
It will even install safely side-by-side with the official released version (just don't turn them both "on" at once)
![image](https://user-images.githubusercontent.com/9258768/101544679-fea7f180-399d-11eb-849e-4a746c0c5498.png)

**Comment by theolaa** [comment]
Ooh, I didn't know you had a bug fixes release, I'll check it out cuz I've been dying to export some models I've been working on!

**Comment by smigglethesniper** [comment]
Sorry if I'm being stupid, but I'm not sure how to import the .mesh files
<img width="195" alt="Screen Shot 2021-01-25 at 16 45 36" src="https://user-images.githubusercontent.com/54944292/105736566-d9ab1600-5f2c-11eb-8092-bcaf054a86c2.png">

**Comment by ross-g** [comment]
Fixed in 0.8 release. (364800cff10bdc822429ec73149cadcb05ef9710)
